extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -85,29 +85,11 @@ public class DefaultAccessLogReceiver implements AccessLogReceiver, Runnable, Cl
     private final boolean rotate;
     private final LogFileHeaderGenerator fileHeaderGenerator;
 
-    public DefaultAccessLogReceiver(final Executor logWriteExecutor, final File outputDirectory, final String logBaseName) {
-        this(logWriteExecutor, outputDirectory.toPath(), logBaseName, null);
-    }
-
-    public DefaultAccessLogReceiver(final Executor logWriteExecutor, final File outputDirectory, final String logBaseName,
-            final String logNameSuffix) {
-        this(logWriteExecutor, outputDirectory.toPath(), logBaseName, logNameSuffix, true);
-    }
-
     public DefaultAccessLogReceiver(final Executor logWriteExecutor, final File outputDirectory, final String logBaseName,
```

Comment:
it seems, all are unused. sorry 

<img width="1447" alt="image" src="https://github.com/user-attachments/assets/d9d664da-4399-488e-977f-92eabf7ab963" />